src/Client/ClientBase.cpp

```diff
@@ -2844,13 +2844,33 @@ class OptionsAliasParser
 }
 
 
-void ClientBase::parseAndCheckOptions(OptionsDescription & options_description, po::variables_map & options, Arguments & arguments)
+void ClientBase::verboseHelpMessageIfNeeded(OptionsDescription & options_description)
```

Comment:
Where exactly is this method called from?

Reply:
l. 3059. Just pushed a commit regarding it